extensions/wikipedia/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Wikipedia Changelog
 
+## [Recent Searches] - 2023-07-04
+- Added `Recent Searches` inside the `Search Page` command to veiw a list of your previously viewed articles
```

Comment:
```suggestion
- Added `Recent Searches` inside the `Search Page` to show your previously viewed articles
```

Reply:
can't believe I missed that 😅. thanks for the fix!